analysis/README.md

```diff
@@ -0,0 +1,41 @@
+# Distbench analysis script
+
+The distbench analysis tool is a python script developed to transform the raw data coming from running a bunch of tests into different file formats that can be used for other analysis purposes. So this tool only produces text files with certain formats.
+
+## How to use it
+
+The script generates a directory tree with summaries across all the clients and services.
+
+To make use of it, we need to decompress the results file from a test sequence.
+
+```bash
+    gunzip name_of_the_file.pb.gz
+```
+We also need a directory in which to create the directory tree of the summaries.
+```bash
+    mkdir output_directory
+```
+to invoke the script use the follwing command line:
+
+```bash
+    bazel run :results_conversion -- --output_directory=output_directory --input_file=name_of_the_file.pb.gz
```

Comment:
bazel run is not the "normal" way od running binaries unless you are modifying their source.